pyrene/src/components/MultiSelect/MultiSelect.jsx

```diff
@@ -249,6 +258,10 @@ MultiSelect.propTypes = {
    * Displays a loading indicator inside of the input.
    */
   loading: PropTypes.bool,
+  /**
+   * Maximum width of the value label in pixels. Use false to disable a maximum width.
+   */
+  maxValueLabelWidth: PropTypes.oneOfType([PropTypes.number, PropTypes.oneOf([false])]), // eslint-disable-line react/no-unused-prop-types
```

Comment:
I think we should just stick with the normal css values.
none, 10%, 42em, max-content ...

That would also remove the need for false. -> none

https://developer.mozilla.org/en-US/docs/Web/CSS/max-width

Reply:
Thank you for the feedback, that definitely makes sense! I'll update it.